content/control-api/testing-with-postman.textile

```diff
@@ -0,0 +1,86 @@
+---
+title: Testing with Postman
+meta_description: "Control API provides a REST API to allow you to manage your Ably account."
+meta_keywords: "Ably, account management, control, REST API, Control API."
+section: control-api
+index: 30
+jump_to:
+  Help with:
+    - Importing into Postman#importing
+    - Sending a request#send-request
+    - See also#see-also
+---
+
+There are several ways you can try out the API. You can use a command line tool such as "Curl":https://curl.se/ or "HTTPie":https://httpie.io/ to test out various requests. Another way is to use a graphical tool such as Postman or Paw. This topic looks at testing with Postman.
```

Comment:
API -> Control API. Maybe provide the link to Postman here where it is first mentioned, instead of in the next section?